### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ngaut**

### CODE REVIEW

**Comment by c4pt0r** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -54,4 +54,5 @@ TiDB is under the Apache 2.0 license. See the [LICENSE](./LICENSES/LICENSE) file
 
 ## Acknowledgments
 - Thanks [cznic](https://github.com/cznic) for providing some great open source tools.
-- Thanks [GolevelDB](https://github.com/syndtr/goleveldb), [LMDB](https://github.com/LMDB/lmdb), [BoltDB](https://github.com/boltdb/bolt) and [RocksDB](https://github.com/facebook/rocksdb) for their powerful storage engines.
+- Thanks [Xiaomi](https://github.com/XiaoMi/themis) for providing some great open source project.
```
Comment (qiuyesuifeng) [inline README.md]: providing the great open source project

Comment (shenli) [inline README.md]: You use "some", so project ->  projects?

File: README.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ Power TiDB with your most favorite engines. TiDB supports most of the popular st
 
 ## Status
 
-TiDB is at its early age and under heavy development, some of the features mentioned above have not been fully implemented.
+TiDB is at its early age and under heavy development, all of the features mentioned above are fully implemented.
```
Comment (qiuyesuifeng) [inline README.md]: Should we update `Multiple storage engine support` section  to add some content about hbase above?

**Comment by shenli** [comment]
Should we metion TiDB is in alpha status in Readme.md?

**Comment by ngaut** [comment]
PTAL

**Comment by coocood** [comment]
LGTM

**Pull request merged**